apps/journeys-admin/src/components/JourneyList/journeyListData.ts

```diff
@@ -9,5 +13,6 @@ export const defaultJourney: Journey = {
   themeName: ThemeName.base,
   themeMode: ThemeMode.light,
   slug: 'default',
-  locale: 'en_US'
+  locale: 'en_US',
+  status: JourneyStatus.published
```

Comment:
In another PR we made this draft

```suggestion
  status: JourneyStatus.draft
```